var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -272,6 +273,8 @@ class Trilinos(CMakePackage, CudaPackage):
     conflicts('+stokhos', when='%xl')
     conflicts('+stokhos', when='%xl_r')
 
+    conflicts('+x11', when='~exodus', msg='+x11 only needed when +exodus')
```

Comment:
I'd suggest removing this conflict. If the user for some reason specifies `+x11` without wanting `+exodus` (so they did it explicitly, but it is useless), then it is probable better to just build it than to throw an error.